src/SlackHookHandler.ts

```diff
@@ -104,7 +104,7 @@ export class SlackHookHandler extends BaseSlackHandler {
                         this.eventHandler.onVerifyUrl(eventPayload.challenge!, eventsResponse);
                         // if RTM is enabled, don't handle this event. We should have assigned a RTM client to the team.
                     } else if (eventPayload.type === "event_callback" &&
-                               this.main.teamIsUsingRtm(eventPayload.team_id)
+                               !this.main.teamIsUsingRtm(eventPayload.team_id)
```

Comment:
This took a few extra seconds to read - is the use of RTM clients for teams documented/commented anywhere? Not clear why we only handle events in the absence of RTM.

Reply:
Ah, will document.

Reply:
I've documented, but to explain it here as well. RTM is an alternative to the events API that pushes the same data, but down websockets. We've left the events API code in for legacy reasons, but that means we need to ensure we do not handle both events coming in from the HookHandler and the RTMHandler for the same team.

Reply:
Improved comments are good + clear, thanks